listeners.js

```diff
@@ -237,5 +242,19 @@ module.exports = function startListeners(ssb, config, onError) {
         }
       )
     )
+
+    // if we find an exclude and it's not excluding us but we don't find a new epoch, even after a while, then create a new epoch, since we assume that the excluder crashed or something
+    pull(
+      ssb.db.query(
+        where(and(isDecrypted('box2'), type('group/exclude-member'))),
+        dbLive({ old: true }),
+        toPullStream()
+      ),
+      pull.filter(isExcludeMember),
+      pull.map((msg) => msg.value.content),
+      pull.filter((content) => !content.excluded.includes(myRoot.id))
```

Comment:
this only works if we assume someone excluding only publishes 1 exclude message when excluding. if they published 2 then one of them would slip through here and we'd try to create a new epoch even if we're excluded.

maybe best solution is instead to stream listMembers and check the exclude list there, since i think that function might look for multiple exclude messages (double check)